### PR TITLE
Misc README.md tweaks

### PR DESCRIPTION
**Pull request by atesgoral**
* Updated links
* Standardized on official spelling "ESLint"
* Removed bit about default configuration (http://eslint.org/docs/user-guide/migrating-to-1.0.0#all-rules-off-by-default)
* Fixed a typo
* ~~Fixed broken coverage badge~~

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# gulp-eslint [![Build Status](https://travis-ci.org/adametry/gulp-eslint.svg)](https://travis-ci.org/adametry/gulp-eslint) [![Coverage Status](https://img.shields.io/coveralls/adametry/gulp-eslint.svg)](https://coveralls.io/r/adametry/gulp-eslint)
```
Comment (shinnn) [inline README.md]: No need to change this. This badge is currently not broken.

According to my own experience the official coverall badge seems to be cached too long and doesn't reflect the latest status immediately.

Comment (atesgoral) [inline README.md]: OK it looks fixed now. It was broken yesterday for whatever reason, so I ignorantly assumed it was an outdated badge markup. Reverted that commit.

**Comment by shinnn** [comment]
One change per commit is much appreciated. It would result in these 3 commits:

1. Update links
2. Standardize on official spelling "ESLint"
3. Remove bit about default configuration
4. Fix typo

**Comment by atesgoral** [comment]
I just went in to fix a typo I spotted and then got carried away :) I normally keep commits separate when I have a plan to start with. I'll see if I can find the time to split these into separate commits later.

**Comment by shinnn** [comment]
> I'll see if I can find the time to split these into separate commits later.

Thanks! Take your time :coffee:

**Comment by ilanbiala** [comment]
Merge?

**Comment by shinnn** [comment]
@ilanbiala https://github.com/adametry/gulp-eslint/pull/94#issuecomment-138983272

**Comment by ilanbiala** [comment]
@shinnn if I create a new PR will you merge that in?

**Comment by shinnn** [comment]
I'll wait for this PR to be updated one more day.

// @atesgoral

**Pull request closed without merging**